.gitignore

```diff
@@ -4,6 +4,7 @@
 !vendor/**
 
 .vscode
+/.idea
```

Comment:
![](https://thumbs.gfycat.com/WateryActualAtlanticspadefish-max-1mb.gif)

Reply:
c'mon, `.vscode` was already there :grimacing: 

I can add this in a separate PR if you insist :smile: